### PR TITLE
Hungarian language update 05

### PR DESCRIPTION
**Pull request by big4billy**
Next translation group. A bit longer then 400, but not much :-)

### CODE REVIEW

**Comment by big4billy** [comment]
> Hi, @big4billy. Please reduce it to 400 or less changes. I just had to request the same from the Vietnamese translation PR too. This rule needs to be fair and consistent. You can just make a backup file and store those extra 30~ lines in that.

we've done it

**Comment by big4billy** [comment]
Dear @zenseii! We'd keep on doing the Hungarian translation. As you know, we have all the translations offline so "just" need to put them in. Would you review our update and corrections? As far as I know, we corrected all the errors appeared during the first automated checks.

File: files/lang/hu.po [hunk 1]
```diff
@@ -2704,47 +2702,46 @@ msgid "MIDI"
...
 msgid "Build a new ship:"
 msgstr "Új hajó építése:"
 
 msgid "Resource cost:"
-msgstr "Fejlesztési költség:"
+msgstr "Erőforrás költség:"
 
 msgid "Total: "
-msgstr ""
+msgstr "Összesen:"
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: SPACE added

File: files/lang/hu.po [hunk 2]
```diff
@@ -2704,47 +2702,46 @@ msgid "MIDI"
...
 
 msgid "Need: "
-msgstr ""
+msgstr "Szükséges:"
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: SPACE added

File: files/lang/hu.po [hunk 3]
```diff
@@ -2755,58 +2752,56 @@ msgstr "Egy- vagy többszemélyes játék indítása."
...
-#, fuzzy
 msgid "Text Support"
-msgstr "Teleportálás"
+msgstr "Szöveg támogatás"
 
 msgid "Change the language of the game."
-msgstr ""
+msgstr "A játék nyelvének megváltoztatása."
 
 #, fuzzy
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: "fuzzy" removed

File: files/lang/hu.po [hunk 4]
```diff
@@ -2755,58 +2752,56 @@ msgstr "Egy- vagy többszemélyes játék indítása."
...
+msgstr "Játék mentése"
 
 msgid "Save the current game."
-msgstr ""
+msgstr "Aktuális játék mentése."
 
 msgid "Quit"
 msgstr "Kilépés"
 
 #, fuzzy
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: "fuzzy" removed

File: files/lang/hu.po [hunk 5]
```diff
@@ -2851,150 +2846,144 @@ msgstr ""
...
-#, fuzzy
 msgid "Your Funds"
-msgstr "500-999"
+msgstr "Forrásaid"
 
 msgid "Planned Gift"
-msgstr ""
+msgstr "Tervezett ajándék"
 
 #, fuzzy
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: "fuzzy" removed

File: files/lang/hu.po [hunk 6]
```diff
@@ -2851,150 +2846,144 @@ msgstr ""
...
-#, fuzzy
 msgid "%{skill} +1"
-msgstr "%{level} %{skill} információ"
+msgstr "%{skill} +1"
 
 msgid "You have learned %{skill}."
-msgstr ""
+msgstr "%{skill} képességre tettél szert."
 
 #, fuzzy
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: "fuzzy" removed

File: files/lang/hu.po [hunk 7]
```diff
@@ -3007,24 +2996,27 @@ msgid ""
...
 msgstr ""
+"Kérem tekintsd meg nagyszerű portékáinkat. Amennyiben kereskedni kívánsz, "
+"kattints arra az elemre, amit és amire cserélni akarsz."
 
 msgid ""
 "You have received quite a bargain. I expect to make no profit on the deal. "
 "Can I interest you in any of my other wares?"
 msgstr ""
+"Jó alkut kötöttél. Nem várok el nyereséget az üzleten."
+"Érdekel esetleg más áru is?"
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: SPACE added

File: files/lang/hu.po [hunk 8]
```diff
@@ -3353,89 +3319,90 @@ msgid ""
...
+msgstr "Ne mutasd"
 
-#, fuzzy
 msgid "Enemy Speed"
-msgstr "Sebesség"
+msgstr "Ellenfél sebessége"
 
 msgid "Auto Resolve"
-msgstr ""
+msgstr "Automata döntéshozatal"
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Shortened, surely OK

File: files/lang/hu.po [hunk 9]
```diff
@@ -3353,89 +3319,90 @@ msgid ""
...
 
 msgid "Auto, No Spells"
-msgstr ""
+msgstr "Automata döntéshozatal, de nem varázsol"
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Shortened, hope enough

File: files/lang/hu.po [hunk 10]
```diff
@@ -3033,223 +3025,202 @@ msgid "Your Resources"
...
 msgid "(no digging)"
-msgstr ""
+msgstr "(nem ásható)"
 
 msgid "penalty: %{cost}"
-msgstr ""
+msgstr "büntetés: %{cost}"
+
+msgid "Uncharted Territory"
+msgstr "Felderítetlen terület"
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Relocated to the original place

**Comment by big4billy** [comment]
> Hi, @big4billy. I left several comments. Please have a look when you have time.

Hi, @zenseii. I corrected all the asked things.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@big4billy and @anon569 , a bunch of thanks for this update!

**Pull request merged**